code/game/machinery/doors/airlock.dm

```diff
@@ -646,14 +646,12 @@ About the new airlock wires panel:
 			if(user)
 				src.attack_ai(user)
 
-/obj/machinery/door/airlock/CanPass(atom/movable/mover, turf/target, height=0)
-	if(src.isElectrified())
-		if(istype(mover, /obj/item))
-			var/obj/item/i = mover
-			if(i.materials[MAT_METAL])
-				var/datum/effect/system/spark_spread/s = new /datum/effect/system/spark_spread
-				s.set_up(5, 1, src)
-				s.start()
+/obj/machinery/door/airlock/CanPass(obj/item/mover)
```

Comment:
I figured we basically only need to check `CanPass(obj/item/mover)` instead of the rest of it